### PR TITLE
some questions and thoughts on the current code

### PR DESCRIPTION
**Pull request by jmkoni**
I have some thoughts on what's currently in the models. Would really appreciate some further thoughts or explanations.

### CODE REVIEW

File: src/main/java/edu/psu/sweng894/group1/capstone/data/entities/Comment.java [hunk 1]
```diff
@@ -16,6 +16,7 @@
      * The parent, optional, of this comment. If this comment is a reply to a comment, this
      * is non-null. If this comment is attached directly to a poll, this is null.
      */
+    // I think this is premature. We don't need to do threading yet and we should add this when it's needed
```
Comment (Lumunix) [inline src/main/java/edu/psu/sweng894/group1/capstone/data/entities/Comment.java]: This is a tree ID, it has nothing to do with threading, I feel we can leave this in here as when we are ready we can display nested comments using a css proper based on the model value

File: src/main/java/edu/psu/sweng894/group1/capstone/data/entities/Comment.java [hunk 2]
```diff
@@ -25,8 +26,8 @@
```
Comment (Lumunix) [inline src/main/java/edu/psu/sweng894/group1/capstone/data/entities/Comment.java]: I have no problems with the additions to this file

File: src/main/java/edu/psu/sweng894/group1/capstone/data/entities/Notification.java [hunk 3]
```diff
@@ -1,7 +1,10 @@
 package edu.psu.sweng894.group1.capstone.data.entities;
```
Comment (Lumunix) [inline src/main/java/edu/psu/sweng894/group1/capstone/data/entities/Notification.java:2]: We need to save notifications for the DB as you need to associate a notification with a user and a single action can generate multiple notifications

**Review by Lumunix — DISMISSED** [review]
I exaplain a few things in the comments, I see no issues with adding this. This is mostly get/setter and data manipulation logic.

File: src/main/java/edu/psu/sweng894/group1/capstone/data/entities/Comment.java (hunk 1 above)
Comment (jmkoni) [reply src/main/java/edu/psu/sweng894/group1/capstone/data/entities/Comment.java]: So it does have to do with threading (ie nested comments?). I think we should remove it UNTIL we need it.

File: src/main/java/edu/psu/sweng894/group1/capstone/data/entities/Notification.java (hunk 3 above)
Comment (jmkoni) [reply src/main/java/edu/psu/sweng894/group1/capstone/data/entities/Notification.java:2]: Still not a reason to save it to the DB... we can add a mailer and send a notification at that point? Again, premature!

**Review by cvelo99 — APPROVED** [review]

**Comment by cvelo99** [comment]
Try to approve this stuff so we can merge it.

**Review by cvelo99 — APPROVED** [review]
Approving, again.

**Pull request merged**